src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -3290,3 +3290,84 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "FederalState_Thueringen" = "Thüringen";
 
+/* Recycle Bin */
+
+"RecycleBin_title" = "Papierkorb";
+
+"RecycleBin_description" = "Tippen Sie auf ein Element um es wiederherzustellen. Nach 30 Tagen wird ein Element automatisch gelöscht.";
+
+"RecycleBin_deleteAllButtonTitle" = "Alle entfernen";
+
+/* - Vaccination Certificate */
+
+"RecycleBin_VaccinationCertificate_headline" = "Impfzertifikat";
+
+"RecycleBin_VaccinationCertificate_vaccinationCount" = "Impfung %i von %i";
+
+"RecycleBin_VaccinationCertificate_vaccinationDate" = "geimpft am %@";
+
+/* - Test Certificate */
+
+"RecycleBin_TestCertificate_headline" = "Testzertifikat";
+
+"RecycleBin_TestCertificate_pcrTest" = "PCR-Test";
+
+"RecycleBin_TestCertificate_antigenTest" = "Schnelltest";
+
+"RecycleBin_TestCertificate_sampleCollectionDate" = "Probenahme am %@";
+
+/* - Recovery Certificate */
+
+"RecycleBin_RecoveryCertificate_headline" = "Genesenenzertifikat";
+
+"RecycleBin_RecoveryCertificate_validityDate" = "gültig bis %@";
+
+/* - Corona Test */
+
+"RecycleBin_CoronaTest_headline" = "Test";
+
+"RecycleBin_CoronaTest_pcrTest" = "PCR-Test";
+
+"RecycleBin_CoronaTest_antigenTest" = "Schnelltest";
+
+"RecycleBin_CoronaTest_registrationDate" = "registriert am %@";
+
+"RecycleBin_CoronaTest_sampleCollectionDate" = "durchgeführt am %@";
+
+/* - Empty State */
+
+"RecycleBin_EmptyState_title" = "Ihr Papierkorb ist leer";
+
+"RecycleBin_EmptyState_description" = "Wenn Sie Zertifikate entfernen, werden sie in den Papierkorb verschoben und hier angezeigt.";
```

Comment:
Aren't Tests also moved to the "Papierkorb" if they are deleted? Should this be mentioned there?